### ISSUE TITLE
[Task](context): make coypable Id semantic and interactive

### ISSUE THREAD

**Issue by franzheidl** [cobaltcore-dev/aurora-dashboard#1077]
The element to copy an Id of an entity is a stack of `<div>` and `<span>` elements with `onClick` handlers. This is non-semantic markup, an accessibility problem, and not representative of the level of craftsmanship we should strive for. 

<img width="290" height="114" alt="Image" src="https://github.com/user-attachments/assets/7bdc2161-8d03-4889-a424-81c504d5ef63" />

<img width="887" height="359" alt="Image" src="https://github.com/user-attachments/assets/b10f7aa3-e92a-40fb-8109-c79828ab9464" />

The element is not recognized as being interactive nor reachable for assistive technologies.

## Expected Behaviour
Keyboard users should be able to tab to and activate the element; screen readers should announce the element as interactive and confirm when copy succeeds.

## Implementation
Use a natively interactive element instead of bolting on assistive support on a non-assistive solution that should have been natively assistive in the first place.

## Task

- [ ] Use a native interactive html element that naturally appears in the accessibility tree
- [ ] Make sure the element is recognized and reachable by assistive technology
- [ ] Make sure the element is implemented in a way that represents best practices

**Comment by guoda-puidokaite** [comment]
Not done in Object Storage Ceph etc. Please fix **everywhere**. CC: @TilmanHaupt

**Comment by TilmanHaupt** [comment]
checked all occurrences again - i see just services where ContentHeader and with it ClipboardText is not implemented at all.  Details etc all work.

**Comment by franzheidl** [comment]
Sorry, re-opened in error.
